conftrak/test/conftest.py

```diff
@@ -0,0 +1,71 @@
+import pytest
+from conftrak.ignition import start_server
+from doct import Document
+from mock import patch
+import ujson
+import jsonschema
+import uuid
+import sys
+import time as ttime
+import subprocess
+import contextlib
+from conftrak.client.commands import ConfigurationReference
+from conftrak.exceptions import ConfTrakNotFoundException
+from conftrak.ignition import Application
+from conftrak.server.engine import db_connect
+from conftrak.server.utils import ConfTrakException
+from requests.exceptions import RequestException
+from tornado.httpclient import HTTPClient
+
+
+testing_config = {
+    "mongo_uri": "mongodb://localhost",
+    "mongo_host": "localhost",
+    "database": "conftrak_test" + str(uuid.uuid4()),
+    "service_port": 7771,
+    "tzone": "US/Eastern",
+    "log_file_prefix": "",
+}
+
+
+@contextlib.contextmanager
+def conftrak_process():
+    try:
+        ps = subprocess.Popen(
+            [
+                sys.executable,
+                "-c",
+                f"from conftrak.ignition import start_server; start_server(args={testing_config}, testing=True)",
+            ]
+        )
+        ttime.sleep(4)
+        yield ps
+    finally:
+        ps.terminate()
+
+
+@pytest.fixture(scope="session")
+def conftrak_server():
+    with conftrak_process() as conftrak_fixture:
+        yield
+
+
+@pytest.fixture
```

Comment:
I think this was needed by pytest-tornado, which you aren't using any more. So I think you can delete this.

Reply:
removed in https://github.com/NSLS-II/conftrak/pull/7/commits/86ff07d1d874477f583fdd739b7e4f25af82f6a3